integration/helpers/pepr.ts

```diff
@@ -0,0 +1,50 @@
+import { resolve } from "node:path";
+import { cp, readFile, writeFile } from "node:fs/promises";
+import { basename } from "node:path";
+import { Spec, Cmd, Result } from "./cmd";
+import { clone } from "ramda";
+
+const HERE = __dirname;
+
+export async function projectRoot(): Promise<string> {
```

Comment:
I don't think we need to actually export this function since it's only called by `pepr.ts` and `pepr.test.ts`. That probably came about during initial implementation, but based off the rest of the unit tests in `pepr.test.ts`, I think this looks like exposing an implementation-detail of `tgzifyModule()` and `cli()` when do don't need to.

Reply:
I mean, you _could_ look at it that way, but... I actually intended it to be exposed as it's a totally reasonable thing for other helpers to want to do.

If there's no _problem_ with exposing it, I think I'd prefer to leave it as-is.  Rather than thinking of it as an implementation detail, think of `tgzifyModule` and `cli` as higher-level helpers that build off this as a basic, pepr-module-scoped `projectRoot` helper instead.